3rd-party/domain-mapping.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Automattic\Jetpack\Constants\Manager as Constants_Manager;
```

Comment:
I'm not sure about this `*_Manager` pattern. Can we come up with something more concise?

Reply:
Addressed in #12616 - I wonder if `Constants` is good enough, or should we alias to `Jetpack_Constants` to mirror what we had before?

Reply:
Also, this seems to occur for other packages - should we be addressing it there as well?